djstripe/stripe_objects.py

```diff
@@ -1534,22 +1596,29 @@ def status(self):
 
 class StripeInvoiceItem(StripeObject):
     """
-Sometimes you want to add a charge or credit to a customer but only actually charge the customer's
-card at the end of a regular billing cycle. This is useful for combining several charges to
-minimize per-transaction fees or having Stripe tabulate your usage-based billing totals.
-(Source: https://stripe.com/docs/api/python#invoiceitems)
+    Representation of a Stripe InvoiceItem object.
```

Comment:
For all of these initial sentences for class docstrings, use this format:

"A dj-stripe representation of a Stripe {object_name} object"